pycompwa/expertsystem/state/particle.py

```diff
@@ -21,6 +21,41 @@
     get_originating_final_state_edges)
 
 
+__all__ = [
```

Comment:
What do you need that for?

Reply:
Currently, only functions/classes listed under `__all__` are documented in the API

Reply:
I see, when did that change?

Reply:
Hm maybe I'm wrong, in the current documentation, all members are there indeed...
https://compwa.github.io/api/pycompwa.expertsystem.state.particle.html#module-pycompwa.expertsystem.state.particle

Reply:
https://github.com/ComPWA/pycompwa/pull/76/commits/3ca8c5d07333782266f27a2268b5cd318999a16b

Reply:
All right. Another topic is that the existing code is actually not so good, that all of functions are "public" and appear in the api, instead of being private/internal functions.

Reply:
Yeah, but I tried not tackling that in the current PR

Reply:
Yeah just a small remark